### DISCUSSION TITLE
Github Account Suspended.

### DISCUSSION THREAD

**Discussion by imseamoon** [community/community#122095] [Other Feature Feedback, Questions, & Ideas]
### Troubleshoot

How can I get recognized quicker so that my account will be unsuspended.

### Body

I have been working recently with my other account @seamoonpandey, email: hackedasocial@gmail.com till it got suddenly suspended yesterday for some reason (I didn't get any violations email so I am unaware of the violation). How can I unsuspend my account , everything related to the account has been blocked including all the workflows and organizations with my account as the only admin is unmanageable now and all those projects I did in past, I cannot access those too (not even view!). I might get fired from my current job due to this mistake, since I am unable to contribute to the projects development.
I hope I get my original account unsuspende.

**Comment by MuhammadRayyan16** [comment]
Response #1

**Comment by MarcosApodaca** [comment]
Go to the link and generate a ticket, so Github can help you with it, they can also provide you with information about why your account was suspended.

https://support.github.com/contact/account

**Comment by ebndev** [comment]
Hi @imseamoon,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
